CHANGELOG.md

```diff
@@ -1,14 +1,18 @@
 # Change log
 
-All notable changes to this project will be documented in this file. The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org).
+All notable changes to this project will be documented in this file. The format is based on [Keep a Changelog](http://keepachangelog.com/en/0.6.0/) and this project adheres to [Semantic Versioning](http://semver.org).
```

Comment:
http://keepachangelog.com/en/0.6.0/ link doesn't work :-)

Reply:
@pamelaju ,
yes, my bad
I re-generated everything using our amazing auto_release tool and now it should be fixed